### PR TITLE
feat: updating payload v2

### PR DESCRIPTION
**Pull request by johnytiago**

### CODE REVIEW

File: apps/asap-server/test/handlers/users/fetch.test.ts [hunk 1]
```diff
@@ -100,8 +98,8 @@ describe('GET /users', () => {
           Authorization: `Bearer token`,
         },
         queryStringParameters: {
-          take: 8,
-          skip: 8,
+          take: '8',
```
Comment (fampinheiro) [inline apps/asap-server/test/handlers/users/fetch.test.ts:101]: Why a string?

Comment (johnytiago) [reply apps/asap-server/test/handlers/users/fetch.test.ts:101]: QueryStringParameters will never be a string https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/aws-lambda/trigger/api-gateway-proxy.d.ts#L107

Comment (fampinheiro) [reply apps/asap-server/test/handlers/users/fetch.test.ts:101]: If Joi parses it as number I don't see a problem :+1:

Comment (johnytiago) [reply apps/asap-server/test/handlers/users/fetch.test.ts:101]: These are not the queryStringParameters the FE sends us. This are the ones that are presented to the lambda framework after AWS processes them. These will always be strings and it's up to Joi to then convert them to numbers then. Them not being strings is actually wrong.

**Review by fampinheiro — APPROVED** [review]
Assuming the patch works with serverless-offline :)

**Pull request merged**